debian/rules

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/make -f
+
+# See debhelper(7) (uncomment to enable)
+
+# Output every command that modifies files on the build system.
+export DH_VERBOSE = 1
+
+%:
+	dh $@
+
+override_dh_auto_build:
+	python3 -m venv .venv
+	bash -c 'source .venv/bin/activate'
+	.venv/bin/pip install debian/pypi/docopt_ng-0.8.1-py2.py3-none-any.whl
+	.venv/bin/pip install debian/pypi/zipp-3.17.0-py3-none-any.whl
+	.venv/bin/pip install debian/pypi/importlib_metadata-6.11.0-py3-none-any.whl
+	.venv/bin/pip install debian/pypi/lxml-4.9.3-cp311-cp311-manylinux_2_28_x86_64.whl
+	.venv/bin/pip install debian/pypi/typing_extensions-4.9.0-py3-none-any.whl
+	.venv/bin/pip install debian/pypi/rnginline-1.0.0-py3-none-any.whl
```

Comment:
Is it possible to install these from PyPI (remote)? If not, could they be stored in the OBS repo instead of here?